src/App.jsx

```diff
@@ -16,7 +16,7 @@ function App() {
       <BrowserRouter>
       <Nav/>
         <Routes>
-          <Route path='/' element={<Home />}></Route>
+          <Route path='/' element={<Home articles={articles} setArticles={setArticles}/>}></Route>
```

Comment:
As the state for articles is declared in app, and we are passing it down straight away, then making two separate api requests using the same function to populate it - it would make sense to either have two separate states within those components and populate those rather than passing the state down from app, or make the request in app and populate the state first before passing it down to both components. The latter option would mean only one api request is made rather than two.

Having said that, If home and articles are displaying the same thing, I think it makes sense to just have the home route though, and have the articles there - you don't have to have a separate page for it! that way we only need one state in home, and one api request to populate it and we dont need to be making any requests inside App